packages/exit-app/package.json

```diff
@@ -22,7 +22,7 @@
     "axios-cache-adapter": "^2.7.3",
     "bnc-onboard": "^1.35.4",
     "classnames": "^2.3.1",
-    "ethcall": "^4.0.7",
```

Comment:
Can we remove this? it looks like we are still using ethcall in `src/store/main/actions.ts`

Reply:
The intention was to swap it completely, I missed that file, I had the library locally that's why it was still building for me, sorry.

I've fixed this in this commit https://github.com/gnosis/zodiac-module-exit/pull/61/commits/5c6b8cfe1265133c5643121947dc07a17d9910f2 🚀